docs/source/platform-features/message-bus/vip/vip-known-identities.rst

```diff
@@ -7,25 +7,46 @@ VIP Known Identities
+| volttron.central       | Multi-Network         | The default identity for a VolttronCentralAgent.  The PlatformAgent by default will use this as it's manager, but can be overridden in the configuration file of individual agents                                                                                 |
++------------------------+-----------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| platform.historian     | Platform              | An individual platform may have many historians available to it, however the only one that will be available through Volttron Central by default will be called this. Note that this does not require a specific type of historian, just that it's VIP Identity    |
++------------------------+-----------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| platform.topic_watcher | Platform              | Agent which publishes alerts for topics based on timing thresholds                                                                                                                                                                                                 |
++------------------------+-----------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| platform.sysmon        | Platform              | Agent which publishes System Monitoring statistics                                                                                                                                                                                                                 |
++------------------------+-----------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| platform.emailer       | Agents                | Agent used by other agents on the platform to send email notifications                                                                                                                                                                                             |
```

Comment:
I am not sure I like the sphere of influence column all together.  Where did that come from?